privacyidea/lib/subscriptions.py

```diff
@@ -254,7 +254,7 @@ def check_subscription(application, max_free_subscriptions=None):
             else:
                 # subscription is still valid, so check the signature.
                 check_signature(subscription)
-                if active_tokens > subscription.get("num_tokens"):
+                if token_users > subscription.get("num_tokens"):
                     # subscription is exceeded
                     raise SubscriptionError(description="Too many tokens "
```

Comment:
The error message might be misleading in this case.

Reply:
@plettich I gave it a 2nd thought. An I think the text is not so bad. The correct text would be 

"Tokens enrolled to too many users".

But would you understand this? A text like

"There are too many users with an active token in your system."

Sounds a bit awkward. But I am open for suggestions!

Reply:
@plettich Please take a look at my new text!